be/src/runtime/lake_tablets_channel.cpp

```diff
@@ -306,23 +343,22 @@ void LakeTabletsChannel::add_chunk(Chunk* chunk, const PTabletWriterAddChunkRequ
             count_down_latch.count_down();
             continue;
         }
+        total_row_num += size;
         int64_t tablet_id = tablet_ids[row_indexes[from]];
         auto& dw = _delta_writers[tablet_id];
         DCHECK(dw != nullptr);
 
         // back pressure OlapTableSink since there are too many memtables need to flush
         while (dw->queueing_memtable_num() >= config::max_queueing_memtable_per_tablet) {
-            auto t1 = std::chrono::steady_clock::now();
-            if (std::chrono::duration_cast<std::chrono::milliseconds>(t1 - t0).count() > request.timeout_ms()) {
+            auto t1 = watch.elapsed_time();
+            if (watch.elapsed_time() / 1000000 > request.timeout_ms()) {
                 LOG(INFO) << "LakeTabletsChannel txn_id: " << _txn_id << " load_id: " << print_id(request.id())
                           << " wait tablet " << tablet_id << " flush memtable " << request.timeout_ms()
                           << "ms still has queueing num " << dw->queueing_memtable_num();
                 break;
             }
             bthread_usleep(10000); // 10ms
-            wait_memtable_flush_time_us +=
-                    std::chrono::duration_cast<std::chrono::microseconds>(std::chrono::steady_clock::now() - t1)
-                            .count();
+            wait_memtable_flush_time_ns += watch.elapsed_time() - t1;
```

Comment:
```suggestion
            wait_memtable_flush_time_us += 10000;
```

Reply:
t1 can be removed.

Reply:
also use stop watch in local tablets channel?

Reply:
Fixed